code/modules/admin/topic.dm

```diff
@@ -2801,7 +2801,8 @@
 
 	else if(href_list["secretsfun"])
 		if(!check_rights(R_SERVER|R_EVENT))	return
```

Comment:
I know you didn't touch this, but would you mind making this compliant with our standards?